src/utils/AuthService.js

```diff
@@ -27,33 +27,47 @@ export default class AuthService {
 
   static logout() {
     // Clear access token and ID token from local storage
-    window.localStorage.removeItem(ACCESS_TOKEN);
-    window.localStorage.removeItem(ID_TOKEN);
-    window.localStorage.removeItem(EXPIRES_AT);
-    window.localStorage.removeItme(PROFILE);
+    return new Promise((resolve) => {
+      window.localStorage.removeItem(ACCESS_TOKEN);
+      window.localStorage.removeItem(ID_TOKEN);
+      window.localStorage.removeItem(EXPIRES_AT);
+      window.localStorage.removeItem(PROFILE);
+      resolve(true);
+    });
   }
 
   static isAuthenticated() {
-    const token = AuthService.getIdToken();
-    return !!token && !AuthService.isTokenExpired(token);
+    const token = this.getIdToken();
```

Comment:
`this` isn't binded.

Reply:
d'oh! you are right, i forgot to do `this.constructor.methodName`

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/static